index.html

```diff
@@ -142,53 +183,58 @@ <h2 class="about-sub-head">Let's Connect</h2>
                         <li><a href="#"><img src="icon/git.svg"></a></li>
                     </ul>
                 </div>
-                     <button class="resume-btn">Get My resume</button>
+                   <button class="resume-btn">Get My resume</button>
```

Comment:
- [ ] Also it seems the changes you made to the `about me` section as requested by the previous reviewer introduced a new issue, On the desktop screen between `764px` and `1235px` the content of this section tends to overflow the width of the page, Please look into this. 
- - Screenshot for reference:
![Screenshot 2023-04-01 211207](https://user-images.githubusercontent.com/107632780/229312262-99b13b0f-e36a-423f-9068-5f2cc71e4249.png)

Also, I'll advise you always try to go through you UI after making changes because it seems you always end up with a new error after fixing the initial one.